pkgs/development/libraries/libui/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit, cmake, pkgconfig, gtk3 }:
+
+stdenv.mkDerivation rec {
+  version = "3.1.a";
+  name = "libui-${version}";
+  src  = fetchgit {
+    url    = "https://github.com/andlabs/libui.git";
+    rev    = "6ebdc96b93273c3cedf81159e7843025caa83058";
+    sha256 = "1lpbfa298c61aarlzgp7vghrmxg1274pzxh1j9isv8x758gk6mfn";
+  };
+
+  buildInputs = [ cmake pkgconfig gtk3 ];
```

Comment:
@FRidh The build produces a shared library with `gtk3` dynamically linked. 

Here you can see `gtk3` in the test program built using the library:
```bash
$ ldd test | grep gtk
        libgtk-3.so.0 => /nix/store/faxfzshl9qw4677cy4v2yk1ah8g649kh-gtk+3-3.18.5/lib/libgtk-3.so.0 (0x00007fa67abc6000)
        libgdk-3.so.0 => /nix/store/faxfzshl9qw4677cy4v2yk1ah8g649kh-gtk+3-3.18.5/lib/libgdk-3.so.0 (0x00007fa67a906000)
```

Do I need to add `gtk3` to `propagatedBuildInputs`?